src/Consolonia.AvaloniaEdit/ConsoleLineNumberMargin.cs

```diff
@@ -0,0 +1,37 @@
+using System.Globalization;
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Media;
+using AvaloniaEdit.Editing;
+using AvaloniaEdit.Rendering;
+
+namespace Consolonia.AvaloniaEdit
+{
+    public class ConsoleLineNumberMargin : LineNumberMargin
```

Comment:
Is the only point of this class is to create FormattedText manually? Can we just override factory for that?